boltdesignsystem/bolt
# DS-660 Ratio Element

## Jira

https://pegadigitalit.atlassian.net/browse/DS-660

## Summary

Ratio Element is created to replace the Ratio Component in the future.

## Details

A new Ratio Element was created and its docs

- schema
- twig template
- scss
- docs

## How to test

Pull the branch. Check if:

- the `schema` is compatible with our standards
- `styles` are efficient and sufficient for this element
- `ratio element` meets use cases and can replace the old `ratio component`
- `docs` are useful and readable

## Release notes

`Ratio Component` will be deprecated and replaced with the new `Ratio Element`.
